ONSdigital/response-operations-ui
# Allow business to view date and time for completed surveys

# Motivation and Context
The business wanted to display the date and time next to a completed survey in the response status pop up. This would reduce the number of calls given to a respondent to get times. 

# What has changed
Able to view response status when complete, and then see date timestamp.

# How to test?
Set a case event to complete of completed by phone and check the response status

# Links
[Trello Card](https://trello.com/c/mr2EipGk/287-us208-int-display-date-and-time-stamp-alongside-rus-ce-response-status-m)
